src/main/java/com/google/api/codegen/metacode/InitCodeNode.java

```diff
@@ -238,16 +237,7 @@ private static InitValueConfig mergeInitValueConfig(
           newSubTrees.add(InitCodeNode.createWithValue(nameString, initValueConfig));
         }
       }
-      // Filter subTrees using fieldSet
```

Comment:
What does this filtering do, and why should it no longer happen?

Reply:
This filtering ensured that **only** the set of fields listed in `context.initFields()` would be given an initialized value. Removing the filtering means that while all fields listed in `context.initFields()` will still get an intialized value, fields that are not listed can also be given initialized values.

The problem this fixes is that `context.initFields()` contains the required fields, but not the optional fields. So even if an optional field is given an sample_code_init_value, it would not be initialized because it will be filtered out.